domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -290,18 +298,31 @@ class TopicListControllerTest {
   }
 
   @Test
-  fun testRetrieveOngoingStoryList_defaultLesson_hasCorrectInfo() {
-    topicListController.getOngoingStoryList(profileId0).toLiveData()
-      .observeForever(mockOngoingStoryListObserver)
+  fun testRetrieveRecommendedActivityList_markChapterCompletedFracStory0Exp0_recommendedStoryListIsCorrect() {
+    storyProgressController.recordCompletedChapter(
+      profileId0,
+      FRACTIONS_TOPIC_ID,
+      FRACTIONS_STORY_ID_0,
+      FRACTIONS_EXPLORATION_ID_0,
+      getCurrentTimestamp()
+    )
     testCoroutineDispatchers.runCurrent()
 
-    verifyGetOngoingStoryListSucceeded()
-    verifyDefaultOngoingStoryListSucceeded()
+    topicListController.getRecommendedActivityList(profileId0).toLiveData()
+      .observeForever(mockRecommendedActivityListObserver)
+    testCoroutineDispatchers.runCurrent()
+
+    verifyGetRecommendedActivityListSucceeded()
+
+    val recommendedActivityList = recommendedActivityListResultCaptor.value.getOrThrow()
+    assertThat(recommendedActivityList.recommendedStoryList.suggestStoryCount).isEqualTo(2)
+    verifyOngoingStoryAsFractionStory0Exploration1(recommendedActivityList.recommendedStoryList.suggestStoryList[0])
+    verifyOngoingStoryAsRatioStory0Exploration0(recommendedActivityList.recommendedStoryList.suggestStoryList[1])
   }
 
   @Test
-  @Ignore("Failing on Circle CI.")
-  fun testRetrieveOngoingStoryList_markRecentlyPlayedFracStory0Exp0_ongoingStoryListIsCorrect() {
+//  @Ignore("Failing on Circle CI.")
```

Comment:
adding a blocker to remove comment before merging

Reply:
Done.